src/ui-config/markets/index.ts

```diff
@@ -28,16 +28,13 @@ export const marketsData: { [key in keyof typeof CustomMarket]: MarketDataType }
       faucet: true,
       governance: true,
       staking: true,
-      collateralRepay: true,
       incentives: true,
     },
     addresses: {
       LENDING_POOL_ADDRESS_PROVIDER: '0x88757f2f99175387ab4c6a4b3067c77a695b0349'.toLowerCase(),
       LENDING_POOL: '0xE0fBa4Fc209b4948668006B2bE61711b7f465bAe',
       WETH_GATEWAY: '0xA61ca04DF33B72b235a8A28CfB535bb7A5271B70',
       FLASH_LIQUIDATION_ADAPTER: '0x9D50F0b23b1805773f607F0B4678d724322B7AC2',
-      REPAY_WITH_COLLATERAL_ADAPTER: '0xf86Be05f535EC2d217E4c6116B3fa147ee5C05A1',
```

Comment:
doesn't work most of the time as there are no fund in the mocked uni v2 - better test this on a fork